### PR TITLE
Change icons in information page

### PR DESCRIPTION
**Pull request by ryugoo**
## Issue
- #75 

## Overview (Required)
- Create social icons at vector drawables
  - Twitter
  - Facebook
  - GitHub
  - YouTube
- Create "DroidKaigi" icon at bitmap image

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by konifar** [comment]
Cool! 🆒

**Comment by konifar** [comment]
> Create "DroidKaigi" icon at bitmap image

This icon is a little complicated. So bitmap is better 😉

**Comment by konifar** [comment]
Can you reformat the code in xml files?

File: app/src/main/res/layout/fragment_information.xml [hunk 1]
```diff
@@ -108,7 +108,7 @@
                             android:layout_marginLeft="@dimen/info_social_icon_margin"
                             android:layout_marginStart="@dimen/info_social_icon_margin"
                             android:onClick="@{viewModel::onClickYouTube}"
-                            android:src="@drawable/ic_youtube_36"
+                            app:srcCompat="@drawable/ic_youtube_vector_36"
```
Comment (konifar) [inline app/src/main/res/layout/fragment_information.xml]: I prefer `_vector` suffix. What do you think?

**Comment by konifar** [comment]
Commented! Thank you for contribution!

File: app/src/main/res/layout/fragment_information.xml (hunk 1 above)
Comment (ryugoo) [inline app/src/main/res/layout/fragment_information.xml]: Oops. Surely I think so too. I will fix it.

**Comment by konifar** [comment]
LGTM! Thank you for contribution! 💯

**Pull request merged**